src/main/java/Duke.java

```diff
@@ -1,10 +1,431 @@
+            fileHandle.delete();
+        }
+    }
+    
+    public static String parseTaskToString(Vector <Task> tasks) {
+        String resultList = "";
+        String taskString = "";
+
+        for (int i = 1; i <= tasks.size(); i++) {
+            String taskType = tasks.get(i - 1).getTaskType();
+            String markDone;
+            if (tasks.get(i - 1).getTaskDone()) {
+                markDone = TASK_DONE_FLAG;
+            } else {
+                markDone = TASK_NOT_DONE_FLAG;
+            }
+
+            switch (taskType) {
+            case "ToDo":
+                taskString = "T " + SEPARATOR + " " + markDone + " " + SEPARATOR + " " + tasks.get(i - 1).getTaskTitle() +
+                        System.lineSeparator();
+                resultList += taskString;
+                break;
+
+            case "Deadline":
+                String deadline = tasks.get(i - 1).getTaskDeadline();
+                taskString = "D " + SEPARATOR + " " + markDone + " " + SEPARATOR + " " + tasks.get(i - 1).getTaskTitle() +
+                        " " + SEPARATOR + " " + deadline + System.lineSeparator();
+                resultList += taskString;
+                break;
+
+            case "Event":
+                String eventTime = tasks.get(i - 1).getTaskDate();
+                taskString = "E " + SEPARATOR + " " + markDone + " " + SEPARATOR + " " + tasks.get(i - 1).getTaskTitle() +
+                        " " + SEPARATOR + " " + eventTime + System.lineSeparator();
+                resultList += taskString;
+                break;
+
+            default:
+                break;
+            }
+        }
+        return resultList;
+    }
+
+    
+    
+    
+    
+    
+    public static void printGreet(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke_HTT. \n What can I do for you?");
+    }
+
+    protected static String getTaskMark(String[] inLine) {
+        int inputTaskIndex = Integer.parseInt(inLine[1]);
+        String taskType = tasks.get(inputTaskIndex - 1).getTaskType();
+        String taskMark = "";
+        switch (taskType) {
+        case "ToDo":
+            taskMark = "[T]";
+            break;
+
+        case "Deadline":
+            taskMark = "[D]";
+            break;
+
+        case "Event":
+            taskMark = "[E]";
+            break;
+
+        default:
+            break;
+        }
+        return taskMark;
+    }
+
+    public static void processInput(){   
+        Scanner scan = new Scanner ( System.in ); 
+        String inData = scan.nextLine(); // user line of input
+        String[] inLine = inData.split(" "); // code process line of input into an array
```

Comment:
You can consider specifying your delimiter as a constant variable to minimise the use of magic literals within this block of code.